Electron/src/components/Cards/SongCard/SongCard.tsx

```diff
@@ -63,6 +65,7 @@ export default function SongCard({
         className={`rounded ${styles.card}`}
         onDoubleClick={handleClickSong}
         onContextMenu={handleRightClick}
+        data-testid="song-card"
```

Comment:
All cards will have the same test id :(. We have to find another way of selecting the card